client/javascript/BrokenExternalLinksReport.js

```diff
@@ -25,10 +26,21 @@
         $.ajax({
           url: "admin/externallinks/start",
           async: true,
-          timeout: 3000
-        });
+          timeout: 3000,
+          success: function() {
+            self.poll();
+          },
+          error: function(e) {
+            if (typeof console !== 'undefined') {
+              let message = (e && e.responseText)
```

Comment:
Remove this, though keep the `self.buttonReset(). We don't need to worry about leaving nice console messages, normal users don't look there. The network panel will include the failed network request includes the response text for developers to look at

Reply:
Updated